frontend/src/main.ts

```diff
@@ -23,4 +25,7 @@ app.use(ConfirmationService);
 app.use(PrimeVue);
 
 app.use(router).mount('#app');
+
+app.config.errorHandler = (err) => onError(err);
```

Comment:
um.... I'm not so sure if we need this or not, this is like a global error handling, but we have it previously in the Toast component, so I still keep I here.